{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/terraform/prod/training-job.tf

```diff
@@ -51,12 +51,6 @@ resource "databricks_job" "model_training_job" {
       base_parameters = {
         env             = local.env
         experiment_name = databricks_mlflow_experiment.experiment.name
-        # Run mode for model validation. Possible values are :
-        #   disabled : Do not run the model validation notebook.
-        #   dry_run  : Run the model validation notebook. Ignore failed model validation rules and proceed to move model to Production stage.
-        #   enabled  : Run the model validation notebook. Move model to Production stage only if all model validation rules are passing.
-        # Please complete the TODO sessions in notebooks/ModelValidation before enabling model validation
-        run_mode = "disabled"
```

Comment:
Why did we move `run_mode` out of terraform and hard code in the python code?

Reply:
It's to keep all parameters in one place. But looks like the staging and prod workspace now share the same set of parameters.
I'm now thinking it may be better to have two input files or at least have two parameters for run_mode in staging and prod workspace.